app/javascript/controllers/conditional_question_controller.js

```diff
@@ -24,4 +36,18 @@ export default class extends Controller {
       })
     }
   }
+
+  displayConditionalNumeric() {
+    let value = this.element.value
+    let conditional_for = JSON.parse(this.element.dataset.info)
+
+    Object.entries(conditional_for).forEach(([key, values]) => {
+      let div = document.getElementById(key + "_div")
+      if(eval((value + values))) {
```

Comment:
Just by reading this I'm not entirely sure what's going on in this line. Seems extreme, but maybe extracting it and naming it for readability would be good?

Reply:
Way better!